core/services/chainlink/application.go

```diff
@@ -404,11 +404,13 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 		globalLogger.Debug("Off-chain reporting v2 disabled")
 	}
 
+	healthChecker := services.NewChecker()
+
 	var lbs []utils.DependentAwaiter
 	for _, c := range legacyEVMChains.Slice() {
 		lbs = append(lbs, c.LogBroadcaster())
 	}
-	jobSpawner := job.NewSpawner(jobORM, cfg.Database(), delegates, db, globalLogger, lbs)
+	jobSpawner := job.NewSpawner(jobORM, cfg.Database(), healthChecker, delegates, db, globalLogger, lbs)
```

Comment:
Could we potentially move `services.NewChecker()` inside of `NewSpawner` so that consumers don't have to initialize and then pass it in?

Reply:
If it were the only usage, yeah, but it is used from this higher level also.